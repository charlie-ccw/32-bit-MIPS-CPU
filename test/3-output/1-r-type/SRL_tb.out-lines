register_v0_final 13df40da
