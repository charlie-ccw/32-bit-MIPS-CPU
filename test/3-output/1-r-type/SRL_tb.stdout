RAM : INIT : Loading RAM contents from test/1-binary/1-r-type/srl_tb.txt
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x8c0b0000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x9efa06d3
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x006b10c2
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x006b10c2
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x00000008
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x9efa06d3
register_v0_final 13df40da
test/bus_cpu_tb.v:52: $finish called at 310 (1s)
